src/model_signing/hashing.py

```diff
@@ -140,14 +140,37 @@ def __init__(self):
         self._ignore_git_paths = True
         self.use_file_serialization()
 
-    def hash(self, model_path: PathLike) -> manifest.Manifest:
-        """Hashes a model using the current configuration."""
+    def hash(
+        self,
+        model_path: PathLike,
+        add_ignored_paths: Iterable[pathlib.Path] = frozenset(),
+    ) -> manifest.Manifest:
```

Comment:
this `hashing.Config` class already has a `set_ignored_paths` method.

Reply:
I think what I would need is an option `add_ignored_paths()` that merge those from the manifest with the ones provided via command line.